pkg/fleet/installer/default_packages.go

```diff
@@ -33,6 +33,7 @@ var PackagesList = []Package{
 	{Name: "datadog-apm-library-dotnet", version: apmLanguageVersion, released: true, condition: apmLanguageEnabled},
 	{Name: "datadog-apm-library-python", version: apmLanguageVersion, released: true, condition: apmLanguageEnabled},
 	{Name: "datadog-apm-library-php", version: apmLanguageVersion, released: true, condition: apmLanguageEnabled},
+	{Name: "datadog-apm-library-go-ebpf", version: apmLanguageVersion, released: true, condition: apmLanguageEnabled},
```

Comment:
This will install the package every time a user sets `DD_APM_INSTRUMENTATION_ENABLED=true`. While this is in dev I'd recommend setting `released` to `false` and either:
- Set `releasedBySite: []string{"datad0g.com"}`
- Use an override in your tests or the install script, with `DD_INSTALLER_DEFAULT_PKG_INSTALL_DATADOG_APM_LIBRARY_GO_EBPF=true`